src/scss/views/widgets/_slider.scss

```diff
@@ -0,0 +1,39 @@
+/*
+ * Here we import the base styles from rc-slider module, and we overwrite/extend
+ * what we need to, to make it look and feel how we want.
+ */
+@import "../../node_modules/rc-slider/assets/index.css";
```

Comment:
neat

Reply:
until they change the path, but yeah, pretty neat lol